src/view/drag-handle/sensor/create-touch-sensor.js

```diff
@@ -23,12 +23,77 @@ type TouchWithForce = Touch & {
   force: number
 }
 
+type WebkitHack = {|
+  preventTouchMove: () => void,
+  releaseTouchMove: () => void,
+|}
+
 export const timeForLongPress: number = 150;
 export const forcePressThreshold: number = 0.15;
 const touchStartMarshal: EventMarshal = createEventMarshal();
-
 const noop = (): void => { };
 
+// Webkit does not allow event.preventDefault() in dynamically added handlers
+// So we add an always listening event handler to get around this :(
+// webkit bug: https://bugs.webkit.org/show_bug.cgi?id=184250
+const webkitHack: WebkitHack = (() => {
+  // Do nothing when server side rendering
+  if (typeof window === 'undefined') {
+    return { preventTouchMove: noop, releaseTouchMove: noop };
+  }
+
+  const shouldBlock: boolean = (() => {
+    // All browsers on iPhone or iPad
+    const target: RegExp = /OS\s11_\d\slike\sMac\sOS\sX/g;
```

Comment:
As discussed with @seancurtis, I will remove this check. This will match against iphones and ipads - but not other webkit based browsers. Given that the cost of this is a single event listener I am okay to add it redundantly to other browsers which do not strictly need it in order to avoid other webkit browsers slipping through the cracks